projects/ngrx.io/content/guide/store/walkthrough.md

```diff
@@ -74,6 +74,12 @@ The following example more extensively utilizes the key concepts of store to man
 <code-example header="src/app/app.component.ts" path="store-walkthrough/src/app/app.component.ts">
 </code-example>
 
+13. Update tsconfig.json and remove (to fix Type 'null' is not assignable to type):
+
+```
+"strictTemplates": true
+```
+
```

Comment:
This shouldn't be required because a new project should have this enabled by default.
Instead, we should update the `tsconfig` that is used for the example(s).

Reply:
Yes, a new project comes with this setting enabled by default.
To fix and make the project compile without errors, this line should be removed or changed to false.